test/model/tft/test_model.py

```diff
@@ -0,0 +1,37 @@
+# Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+
+import pytest
+
+from gluonts.model.tft import TemporalFusionTransformerEstimator
+
+
+@pytest.fixture()
+def hyperparameters():
+    return dict(ctx="cpu", epochs=1, learning_rate=1e-3,)
+
+
+@pytest.mark.parametrize("hybridize", [True, False])
+def test_accuracy(accuracy_test, hyperparameters, hybridize):
+    hyperparameters.update(num_batches_per_epoch=200, hybridize=hybridize)
+    accuracy_test(
+        TemporalFusionTransformerEstimator, hyperparameters, accuracy=0.2
+    )
```

Comment:
In this test, all other models are run with very minimal settings (like small number of nodes or layers, and short-ish epochs) so as not to take too long. This test here appears to be timing out (we have a 30s timeout on each test case I believe): maybe the settings here need to be adjusted?